k8s-artifacts/mg-adapter/service.yaml

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Service for adapter
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mg-adapter
+  name: adapter-service
+spec:
+  type: NodePort
+  selector:
+    app: mg-adapter
+  ports:
+    - name: "18000"
+      port: 18000
+      targetPort: 18000
+      protocol: TCP
+      nodePort: 30199
```

Comment:
This does not need. a node port

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf